lib/ClangImporter/ImportDecl.cpp

```diff
@@ -3812,7 +3812,13 @@ namespace {
             decl->isVariadic(), isInSystemModule(dc), name, bodyParams);
 
         if (auto *mdecl = dyn_cast<clang::CXXMethodDecl>(decl)) {
-          if (!mdecl->isStatic()) {
+          if (!mdecl->isStatic() &&
```

Comment:
Could you flip this condition to make it easier to read?

Reply:
Done.